analyses/cell-type-ewings/ewings-cell-type.Rproj

```diff
@@ -1,5 +1,4 @@
 Version: 1.0
-ProjectId: be2aaf55-5c6f-4cc1-a54b-0330df4fb278
```

Comment:
Is there a reason not to keep this? As long as you're always working with the current (or newish) RStudio version, this should be stable.

Reply:
No I think this just snuck in by accident. My guess is I need to update Rstudio.